src/main.coffee

```diff
@@ -5,7 +5,15 @@ feedback = require './feedback'
 
 time = -> (new Date()).getTime()
 
-zxcvbn = (password, user_inputs = []) ->
+zxcvbn = (password, options = {}) ->
+  if options instanceof Array
+    user_inputs = options # backward-compatibility
+  else if typeof options === 'object'
+    { user_inputs, feedback_messages } = options
```

Comment:
In that case `user_input` should be initialised with an empty array to mimic the old behaviour. Otherwise an exception is thrown in [main.coffee L20](https://github.com/paulballesty/zxcvbn/blob/ace4b01a3215350f316f21e25f03e7cdbbee6f12/src/main.coffee#L20) when you trying to iterate over the values of an undefined user_input array.